pywebpush/__init__.py

```diff
@@ -382,6 +382,7 @@ def webpush(subscription_info,
             curl=False,
             timeout=None,
             ttl=0,
+            requests_session=None,
```

Comment:
I'd prefer if new arguments are added at the end of the arg list. This is particularly important for languages like Python where you don't always declare the arg when you pass them (e.g there could be a few implementations of `webpush(sub_info, data, key, claims, `aes128gcm`, true, None, 100, False, None)` out there. As it is, this PR would break them. )
Thanks!